### PR TITLE
Add make-release.sh script to automate the release process

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Add make-release.sh script to automate certain steps of release process

### CODE REVIEW

File: make-release.sh [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    lastCommitComment="$(git log -1 --pretty=%B)"
+    hub pull-request -o -f -m "${lastCommitComment}
+${lastCommitComment}" -b "${BRANCH}" -h "${PR_BRANCH}"
+  fi 
+fi
+
+popd > /dev/null || exit
+
+# cleanup tmp dir
+# cd /tmp && rm -fr "$TMP"
```
Comment (nickboldt) [inline make-release.sh]: are you leaving this commented until you've had a chance to run this and know it works ? :)

Comment (mkuznyetsov) [reply make-release.sh]: woops :smile:

File: make-release.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+git checkout "${BASEBRANCH}"
+
+# create new branch off ${BASEBRANCH} (or check out latest commits if branch already exists), then push to origin
+if [[ "${BASEBRANCH}" != "${BRANCH}" ]]; then
+  git branch "${BRANCH}" || git checkout "${BRANCH}" && git pull origin "${BRANCH}"
+  git push origin "${BRANCH}"
+  git fetch origin "${BRANCH}:${BRANCH}"
+  git checkout "${BRANCH}"
+fi
+
```
Comment (nickboldt) [inline make-release.sh:87]: when do you replace 7.9.0-SNAPSHOT with 7.9.0 in the 7.9.x branch?

Comment (mkuznyetsov) [reply make-release.sh:87]: it is done by centos ci job, so this operation is not needed here

Comment (nickboldt) [reply make-release.sh:87]: please add a comment and pointer to the centos ci job/script so it's more clear and in 6mo when we've both forgotten all about this it'll be easier to remember it :)

**Review by nickboldt — APPROVED** [review]
+1 after fixes to comments/doc

**Pull request merged**